internal/db/migrations/2024-08-28.0-tss_transactions.sql

```diff
@@ -0,0 +1,35 @@
+-- +migrate Up
+
+CREATE TABLE tss_transactions (
+    transaction_hash TEXT PRIMARY KEY,
+    transaction_xdr TEXT,
+    webhook_url TEXT,
+    current_status TEXT,
+    created_at TIMESTAMPTZ DEFAULT NOW(),
+    updated_at TIMESTAMPTZ DEFAULT NOW(),
+    claimed_until TIMESTAMPTZ
```

Comment:
Shall we add NOT NULL constraints on the `transaction_hash` (tho it's PK), `transaction_xdr`, `current_status`, `created_at` and `updated_at` columns?

Reply:
PK columns are non null by default but will add NOT NULL to others